### PR TITLE
User interface mockup

### PR DESCRIPTION
**Pull request by jyeray**

### CODE REVIEW

File: src/lib/update-alternatives/UI/alternatives_dialog.rb [hunk 1]
```diff
@@ -22,50 +22,66 @@
 Yast.import "UI"
 
 module UpdateAlternatives
-
+  # List all alternatives for an alternative group.
```
Comment (jreidinger) [inline src/lib/update-alternatives/UI/alternatives_dialog.rb]: ACtually this class is not list all alternatives, it is dialog. It should write something like `Dialog for displaying possible alternatives for one particular group.`?

File: src/lib/update-alternatives/UI/alternatives_dialog.rb [hunk 2]
```diff
@@ -22,50 +22,66 @@
...
       )
     end
 
     def set_handler
-      finish_dialog()
+      finish_dialog
+    end
+
+    def auto_handler
+      log.info("You have use the \"Set automatic mode\" button")
```
Comment (jreidinger) [inline src/lib/update-alternatives/UI/alternatives_dialog.rb]: log is for developer and not for "you" so it should write something like `User selected "Set automatic mode" button`

File: src/lib/update-alternatives/UI/main_dialog.rb [hunk 3]
```diff
@@ -23,36 +23,44 @@
 Yast.import "UI"
 
 module UpdateAlternatives
-
+  # Dialog where all alternatives groups in the system are listed.
```
Comment (jreidinger) [inline src/lib/update-alternatives/UI/main_dialog.rb:26]: :+1:

File: src/lib/update-alternatives/UI/main_dialog.rb [hunk 4]
```diff
@@ -23,36 +23,44 @@
...
+        footer
       )
     end
 
     def show_handler
       AlternativesDialog.new.run
     end
 
+    def alternatives_table_handler
+      AlternativesDialog.new.run
```
Comment (jreidinger) [inline src/lib/update-alternatives/UI/main_dialog.rb:44]: so change of item in table immediatelly open dialog? I am not sure if it is good idea and also a bit inconsistent with other modules ,where you select item and then action ( clicking on button ).

File: src/lib/update-alternatives/UI/alternatives_dialog.rb (hunk 1 above)
Comment (jyeray) [inline src/lib/update-alternatives/UI/alternatives_dialog.rb]: You're right, it is better to understand on that way.

File: src/lib/update-alternatives/UI/main_dialog.rb (hunk 4 above)
Comment (jyeray) [inline src/lib/update-alternatives/UI/main_dialog.rb:44]: This is called when the user double click on an item of the table. Anyway, I remove it?

Comment (jreidinger) [inline src/lib/update-alternatives/UI/main_dialog.rb:44]: ok, for double click it make sense.

**Comment by jreidinger** [comment]
LGTM, thanks

**Pull request merged**